magicctapipe/scripts/lst1_magic/lst_m1_m2_cloud_correction.py

```diff
@@ -0,0 +1,417 @@
+                        "intensity": hillas_params.intensity,
+                        "x": hillas_params.x,
+                        "y": hillas_params.y,
+                        "r": hillas_params.r,
+                        "phi": hillas_params.phi,
+                        "length": hillas_params.length,
+                        "width": hillas_params.width,
+                        "psi": hillas_params.psi,
+                        "skewness": hillas_params.skewness,
+                        "kurtosis": hillas_params.kurtosis,
+                        "slope": timing_params.slope,
+                        "intercept": timing_params.intercept,
+                        "deviation": timing_params.deviation,
+                        "intensity_width_1": leakage_params.intensity_width_1,
+                        "intensity_width_2": leakage_params.intensity_width_2,
+                        "pixels_width_1": leakage_params.pixels_width_1,
+                        "pixels_width_2": leakage_params.pixels_width_2,
+                        "conc_cog": conc_params.cog,
+                        "conc_core": conc_params.core,
+                        "conc_pixel": conc_params.pixel,
+                        "impact": impact,
+                        "pointing_alt": pointing_alt,
+                        "pointing_az": pointing_az,
+                        "alt": alt_rad,
+                        "az": az_rad,
+                        "wl": wl,
+                        "log_intensity": log_intensity,
+                        "timestamp": timestamp,
+                        "time_diff": time_diff,
+                        "zenith": zenith,
+                        "n_islands": n_islands,
+                        "n_pixels": n_pixels,
+                        "disp_dx": disp_dx,
+                        "disp_dy": disp_dy,
+                        "disp_norm": disp_norm,
+                        "disp_angle": disp_angle,
+                        "disp_sign": disp_sign,
+                        "src_x": src_x,
+                        "src_y": src_y,
+                    }
+                )
+
+    df = pd.DataFrame(data)
+    return df
+
+
+def main():
+    """Main function."""
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--input_file",
+        "-i",
+        dest="input_file",
+        type=str,
+        required=True,
+        help="Path to an input .h5 DL1 data file",
+    )
+
+    parser.add_argument(
+        "--output_file",
+        "-o",
+        dest="output_file",
+        type=str,
+        default="./data",
+        help="Path to a directory where to save an output corrected DL1 file",
+    )
+
+    args = parser.parse_args()
+
+    subarray_info = SubarrayDescription.from_hdf(args.input_file)
+
+    tel_descriptions = subarray_info.tel
+    camgeom = {}
+    for telid, telescope in tel_descriptions.items():
+        camgeom[telid] = telescope.camera.geometry  # 1 - LSTCam; 2,3 - MAGICCam
```

Comment:
the comment: "# 1 - LSTCam; 2,3 - MAGICCam" is only valid for MAGIC+LST1 data

Reply:
This was my working comment - I deleted it.